src/App.jsx

```diff
@@ -14,57 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const clearSelection = () => {
+    setSelectedGood('');
```

Comment:
Consider setting `selectedGood` to `null` instead of an empty string to better represent the absence of a selection. This can help avoid potential issues with type checking or comparisons elsewhere in the code.

Reply:
agree